.changeset/proud-cycles-roll.md

```diff
@@ -0,0 +1,14 @@
+---
+'@clerk/clerk-js': minor
```

Comment:
Also thought of bumping clerk-js as a minor release, but leaving the others as patches, with the reasoning that the main functionally is on clerk-ks and the other packages are just patching on top of it